qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -0,0 +1,171 @@
+import os
+import re
+import sys
+import shutil
+import argparse
+import subprocess
+
+
+# Get dpinst.exe
+def getdpinst(vol_utils):
+    if os.environ.get('PROCESSOR_ARCHITECTURE') == "AMD64":
+        dpinst_dir = "%s\dpinst_64.exe" % vol_utils
+    else:
+        dpinst_dir = "%s\dpinst_32.exe" % vol_utils
+    if not os.path.exists("C:\dpinst.exe"):
+        shutil.copy(dpinst_dir, "C:\dpinst.exe")
+    else:
+        print "dpinst.exe is already existed"
+    if not os.path.exists("C:\dpinst.xml"):
+        dpinst_xml = "%s\dpinst.xml" % vol_utils
+        shutil.copy(dpinst_xml, "C:\dpinst.xml")
+    else:
+        print "dpinst.xml is already existed"
+
+
+# Install certificate
+def certutil(vol_utils):
+    certutil_cmd = "certutil -addstore -f TrustedPublisher %s\\redhat.cer" % vol_utils
+    if not os.path.exists("C:\Windows\system32\certutil.exe"):
+        shutil.copy("%s\certutil.exe" % vol_utils, "C:\Windows\system32")

```

Comment:
I'm not sure that we always have permission to copy file in  system root directory.